client/yarn.lock

```diff
@@ -2676,6 +2676,13 @@
   "resolved" "https://registry.npmjs.org/axe-core/-/axe-core-4.3.2.tgz"
   "version" "4.3.2"
 
+"axios@^0.21.1":
```

Comment:
Shouldn't make any difference. But maybe this should this be inside npm, rather than yarn